testing/e2e/manifests/defaultBackendClientServer.go

```diff
@@ -0,0 +1,228 @@
+package manifests
+
+import (
+	_ "embed"
+	"github.com/Azure/aks-app-routing-operator/api/v1alpha1"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:embed embedded/defaultBackendClient.go
+var dbeClientContents string
+
+//go:embed embedded/defaultBackendServer.go
+var dbeServerContents string
+
+type DefaultBackendResources struct {
+	Client                 *appsv1.Deployment
+	Server                 *appsv1.Deployment
+	Ingress                *netv1.Ingress
+	DefaultBackendServer   *appsv1.Deployment
+	Service                *corev1.Service
+	DefaultBackendService  *corev1.Service
+	NginxIngressController *v1alpha1.NginxIngressController
+}
+
+func (t DefaultBackendResources) Objects() []client.Object {
+	ret := []client.Object{
+		t.Client,
+		t.Server,
+		t.Service,
+		t.Ingress,
+		t.DefaultBackendServer,
+		t.DefaultBackendService,
+		t.NginxIngressController,
+	}
+
+	for _, obj := range ret {
+		setGroupKindVersion(obj)
+	}
+
+	return ret
+}
+
+func DefaultBackendClientAndServer(namespace, name, nameserver, keyvaultURI, host, tlsHost string) DefaultBackendResources {
+	name = nonAlphanumericRegex.ReplaceAllString(name, "")
+
+	// Client deployment
+	clientDeployment := newGoDeployment(dbeClientContents, namespace, name+"-client")
+	clientDeployment.Spec.Template.Annotations["openservicemesh.io/sidecar-injection"] = "disabled"
```

Comment:
Check on this, see if original tests overwrite the value in OSM specific E2E tests